Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m

```diff
@@ -195,7 +198,7 @@ -(void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
 - (NSString *)tableView:(UITableView *)tableView titleForHeaderInSection:(NSInteger)section
```

Comment:
The search view uses the same tableview delegate methods. Where is this datepicker specific code you are referencing?